ai-gen-tools/earthquake-shakemap-urls/scripts/verify_notebook.py

```
@@ -0,0 +1,60 @@
+import json
+import subprocess
+import sys
+
+
+def main():
+    if len(sys.argv) != 2:
+        print("Usage: python3 verify_notebook.py <path/to/notebook.ipynb>")
+        sys.exit(2)
+
+    path = sys.argv[1]
+    result = subprocess.run(
```

Ruff (0.16.1)

[error] `subprocess` call: check for execution of untrusted input

(S603)